StackUsingLinkedList.java

```diff
@@ -0,0 +1,55 @@
+import java.io.*;
+public class StackUsingLinkedList{
+       public static class Node{
+		          int data;
+		          Node next;
+		   
+		         Node (int d){
+			          data=d;
+			          next=null;
+		          }
+	   }
+	  public static Node top;
+	  
+    public static void push(int x){
```

Comment:
@atulya0622 , should've just called addAtHead... Would've made it easy for u..reusing code is important too